src/components/pages/Dashboard/RenderClientDashboard.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import ClientOptions from '../LoggedInOptions/ClientOptions';
+
+import { Layout, Descriptions, Menu, Image, Avatar } from 'antd';
```

Comment:
Recommend removing 'Descriptions' before we merge this branch

Reply:
Change made